scaleway/resource_rdb_instance_beta.go

```diff
@@ -39,7 +39,6 @@ func resourceScalewayRdbInstanceBeta() *schema.Resource {
 			"is_ha_cluster": {
```

Comment:
This is out of this MR but when reading it this field name seems strange, IMHO `enable_ha_cluster` would make more sense don't you think ?

Reply:
It would make more sense indeed. is_ha_cluster is what is being used currently in the CreateInstance API.

So I am balancing using a better name vs keeping the attribute name in sync with the API and dealing with renaming an attribute without breaking compatibility.

I am more inclined to keep it as is for now, what do you think?

Reply:
IMO it makes more sense to write `enable_ha_cluster = true` in the HCL:`is_ha_cluster` is the read only name of the property through the API, whereas `enable_ha_cluster` is an actionnable attribute in the terraform config.

Reply:
Sounds good, I'll update this in a separate PR.